cvs diff nsFileChannel.cpp (in directory D:\Projects\mozilla\netwerk\protocol\file\src\)
Index: nsFileChannel.cpp
===================================================================
RCS file: /cvsroot/mozilla/netwerk/protocol/file/src/nsFileChannel.cpp,v
retrieving revision 1.92
diff -r1.92 nsFileChannel.cpp
335c335
<         NS_ASSERTION(NS_SUCCEEDED(rv), "RemoveChannel failed");
---
>         NS_ASSERTION(NS_SUCCEEDED(rv2), "RemoveChannel failed");

*****CVS exited normally with code 1*****

